### PR TITLE
Add pyhmc

### PR DESCRIPTION
**Pull request by synapticarbors**
@mpharrigan I'm separating out this recipe from msmbuilder to simplify debugging it. 

cc/ #2054

### CODE REVIEW

**Comment by synapticarbors** [comment]
@conda-forge/staged-recipes The failure on Travis not being able to find numpy seems strange. Does anyone know what might be going on here?

**Comment by ocefpaf** [comment]
No but that is the same failure @jochym is getting in his PR.

**Comment by scopatz** [comment]
It seems like it might be a numpy 1.10 thing.  Try olnly building for numpy 1.11

**Comment by jochym** [comment]
It will probably not work if this is the same issue that I have. I have tried multiple combinations - nothing worked. It seems like openblas/numpy in conda-forge has some problems for PR. I finally managed to build it limiting the build to py27 and numpy 1.11 on osx and linux. At the same time feedstock with quite similar setup and partially shared code build fine on full matrix (also as a PR). It looks like this issue is limited to staged-recipes PRs.

**Comment by synapticarbors** [comment]
For reference, the numpy error is being tracked here:

https://github.com/conda-forge/numpy-feedstock/issues/42

**Comment by mpharrigan** [comment]
Thanks for checking this out @synapticarbors !

**Comment by jakirkham** [comment]
Please enable CircleCI on your fork and please rebase on current `master`.

**Comment by jakirkham** [comment]
Restarted Travis CI and appears that `numpy` issue goes away with `conda-build-all` version `1.0.0`, but is present in `1.0.1`.

File: recipes/pyhmc/add-license.patch [hunk 1]
```diff
@@ -0,0 +1,50 @@
+--- /dev/null	2016-11-22 09:55:59.296008905 -0800
++++ LICENSE	2016-12-08 15:52:45.029377387 -0800
```
Comment (jakirkham) [inline recipes/pyhmc/add-license.patch]: Could we please get the license added to `MANIFEST.in` for future inclusions in the `sdist` when released.

Comment (jakirkham) [inline recipes/pyhmc/add-license.patch]: Should add this is an actionable for @mpharrigan .

**Comment by scopatz** [comment]
Here are the docs for enabling Circle: http://conda-forge.github.io/docs/ciservices.html#circleci-linux

**Comment by jakirkham** [comment]
Ha! We have got CircleCI back at staged-recipes.

cc @conda-forge/core @conda-forge/staged-recipes

**Comment by scopatz** [comment]
How is that?

**Comment by jakirkham** [comment]
Wrote it up in this [comment]( https://github.com/conda-forge/status/issues/30#issuecomment-270449574 ).

**Comment by scopatz** [comment]
Nice find!

**Comment by jakirkham** [comment]
I have been in discussions with CircleCI support since the problem began. They added this setting, but was having trouble getting it to work before. Now it appears to be working fine. 😄

**Comment by scopatz** [comment]
I see, is this something that we can automatically enable for all feedstocks?

**Comment by jakirkham** [comment]
Not that I know of at present. Definitely have been asking about it.

**Comment by scopatz** [comment]
Ok, well we should probably add your status message with the image to the docs then, so maintainers can know how to enable it themselves for the time being.

**Comment by jakirkham** [comment]
Agreed that it should go in the docs. Though I don't know if the average maintainer has access to that setting page or not. If they do, great, but if not then this falls on a very small number of people. In that case, the existing workaround is still good until we can roll out a fix across everything.

File: recipes/pyhmc/add-license.patch (hunk 1 above)
Comment (mpharrigan) [inline recipes/pyhmc/add-license.patch]: https://github.com/rmcgibbo/pyhmc/pull/12

**Comment by isuruf** [comment]
@jakirkham, @scopatz, I can confirm that a maintainer can toggle that setting in projects that he/she has access to. I did it for https://github.com/conda-forge/piranha-feedstock and it worked.

**Comment by jakirkham** [comment]
That's great news! Thanks for letting us know @isuruf . Guess we should add this to the docs then. 😉

**Comment by mpharrigan** [comment]
Superseded by #2054

**Pull request closed without merging**